apps/emqx_authz/src/emqx_authz_api_sources.erl

```diff
@@ -452,21 +452,21 @@ write_cert(#{<<"ssl">> := #{<<"enable">> := true} = SSL} = Source) ->
     CertPath = filename:join([emqx:get_config([node, data_dir]), "certs"]),
     CaCert = case maps:is_key(<<"cacertfile">>, SSL) of
                  true ->
-                     {ok, CaCertFile} = write_file(filename:join([CertPath, "cacert-" ++ emqx_plugin_libs_id:gen() ++".pem"]),
+                     {ok, CaCertFile} = write_file(filename:join([CertPath, "cacert-" ++ emqx_misc:gen_id() ++".pem"]),
```

Comment:
this not only makes the "certs" dir a chaos, but also grave yard for all old certificates.
it's hard to know which one is for which resource,
and hard to know which ones are still in use.

possible to add some correlation info in the file name ?

Reply:
I originally used the ID of the instance as the name of an intermediate directory, but for some reason I deleted it.

Let me add it back, and I forgot to delete these certificate files when I delete the instance, I will implement these in another PR.

Reply:
ok, please remember to follow up with another PR